Counter.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import {Button, Text, View} from 'react-native';
+import styles from './styles';
+
+interface Props {}
+interface State {
+  count: number;
+}
+
+export default class Counter extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { count: 0 };
```

Comment:
Because only state is being set in the constructor it can be removed. Just set the state in the class.
`private state = { count: 0 }`.